docs/requirements.md

```diff
@@ -0,0 +1,129 @@
+# Business requirements
+
+> TODO
+
+# User requirements
+
+## Use cases
+
+### Custom Consumer Chain
+
+> TODO e.g., Quicksilver
+
+### Contract Consumer Chains
+
+> TODO e.g., TBA 
+
+### Governance-Gated CosmWasm Consumer Chains
+
+> TODO e.g., Lido
+
+# Functional requirements
+
+## Assumptions
+
+1. Interchain Security packets will (almost) never timeout.
+
+## Features
+
+### 1 - Bootstrapping
+
+| ID  | Description | Verification | Status | Release |
+| --- | ----------- | ------------ | ------ | ------- |
+| 1.01 | A chain has the ability to enable or disable Interchain Security functionality in the genesis state. | Covered by 1.01.x | See 1.01.x | v1 |
+| 1.01.01 | A chain has the ability to enable or disable Interchain Security **_provider_** functionality in the genesis state. | `None` |  `Not Implemented` | TBA |
+| 1.01.02 | A chain has the ability to enable or disable Interchain Security **_consumer_** functionality in the genesis state. | `None` |  `Implemented` | v1 |
```

Comment:
That's a good point. I've added it since we already have the `Enabled` param on the consumer chain. Probably it's not necessary though. @sainoe Is there a need for this param on the consumer chain?